### DISCUSSION TITLE
Feature Request: Support for the "|" operator

### DISCUSSION THREAD

**Discussion by dberardo-com** [cwida/duckpgq-extension#167] [Ideas]
This feature request appeared here: https://github.com/cwida/duckpgq-extension/discussions/161#discussioncomment-11359415

Bascially it should emulate the Cypher pattern:

"MATCH (wallstreet {title: 'Wall Street'})<-[:ACTED_IN|DIRECTED]-(person)"

see also https://github.com/apache/age/issues/2091

**Comment by dberardo-com** [comment]
quoting a potential workaround proposed by the authors:

> Does the | operator act as an OR statement? If so, that's something I want to implement in the long term, but I will not give guarantees at the moment. I think a workaround would be to create a new table which is a UNION_ALL on the ACTED_IN and DIRECTED tables and use that.

**Comment by Dtenwolde** [comment]
Thanks for the suggestions! Created an issue here #168 to track the progress, but won't give any promises regarding timeline :) Leaving this thread open for potential further discussion
